frontend/src/hooks/collab-room/useGetCurrentAttempt.ts

```diff
@@ -0,0 +1,32 @@
+import { Attempt } from "@/@types/attempt";
+import { QuestionComplexity } from "@/@types/models/question";
+import * as Y from "yjs";
+
+// TODO: create the attempt here
+const useGetCurrentAttempt = (document: Y.Doc | null): Attempt => {
+  // TODO: make use of useshareddocument here to get the attempt id and language, etc
```

Comment:
Should we uncomment it so that the `master` branch tracks languages across users? Otherwise we can wait for attempt to be up before merging this